### PR TITLE
[salt] Better Positional Args Handling + Docs

### PR DESCRIPTION
**Pull request by jcockhren**
### Status 
bugfix

### Description

- Fixed execution module actions that failed to collect values passed to the `module` keyword.
- Improved docs with example of sending multiple positional arguments.
- Better handling of positional arguments when processing action call.

### Checklist (tick everything that applies)

- [ ] Metadata: pack.yaml, icon, structure (required for new packs)
- [x] Version bump (required for changed packs)
- [x] Code linting (required, can be done after the PR checks)
- [x] Tests (not required but really recommended)

### CODE REVIEW

**Comment by Kami** [comment]
Thanks!

Now that we have a simple unit testing framework for packs in place, it wouldn't be such a bad idea to add some tests for the base class when you get a chance - https://docs.stackstorm.com/development/pack_testing.html :)

**Comment by jcockhren** [comment]
@Kami word. Will do. jumping on that now.

**Comment by Kami** [comment]
Great, thanks!

File: packs/salt/tests/test_action_local.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+CONFIG_DATA = {
+    'api_url': 'https://example.com',
+    'username': 'this',
+    'password': 'that'
+}
+requests_mock.Mocker.TEST_PREFIX = 'test'
+
+
+class SaltLocalActionTestCase(testtools.TestCase, BaseActionTestCase):
```
Comment (jcockhren) [inline packs/salt/tests/test_action_local.py]: @Kami I think this is a good approach for pack that use requests, any thoughts?

Comment (Kami) [inline packs/salt/tests/test_action_local.py]: Yeah, looks good to me :)

Although, unless I'm missing something (please correct me if I'm wrong) it doesn't look like you are actually asserting on the HTTP requests so you could simply set `requests.requests = mock.Mock()` or similar and it should still work.

**Comment by Kami** [comment]
Merged into master.

Nice work with the tests and thanks again for the contribution!

File: packs/salt/tests/test_action_local.py (hunk 1 above)
Comment (jcockhren) [inline packs/salt/tests/test_action_local.py]: @Kami yes. without `register_uri` the tests will fail given `action.run` is making a call to the the Salt API. I'm just using the `self.action.data` to very we're sending the right data.

Comment (Kami) [inline packs/salt/tests/test_action_local.py]: Yeah, I think just setting it to mock should also work, but I'm fine with your current approach as well :)

This way in the future we can also assert on the URL parameters and post data if needed :)

**Pull request merged**